Task 06(2)/Task 06(2)/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_06_2_
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string word = "клоун";
+            Console.WriteLine($"Из слова {word}, мы сделали слово {word[0]}{word[3]}{word[1]}{word[2]}{word[4]}");
```

Comment:
Слова лучше бы заключить в кавычки
```suggestion
            Console.WriteLine($"Из слова \"{word}\", мы сделали слово \"{word[0]}{word[3]}{word[1]}{word[2]}{word[4]}\"");
```